liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -112,6 +112,13 @@ protected void installLooseConfigWar(MavenProject proj, LooseConfigData config)
         looseWar.addOutputDir(looseWar.getDocumentRoot(), new File(proj.getBuild().getOutputDirectory()),
                 "/WEB-INF/classes");
 
+        // retrieve the directories defined as resources in the maven war plugin
+        String webResources[] = MavenProjectUtil.getPluginConfiguration(proj, "org.apache.maven.plugins",
+            "maven-war-plugin", "webResources", "resource", "directory");
+        for (int i = 0; i < webResources.length; i++) {
```

Comment:
Should we also check if `webResources != null`?  Since `MavenProjectUtil.getPluginConfiguration` could possibly return null.

Reply:
Agreed